ssl_cert_tracker/db_helper.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+from datetime import datetime
+import time
+import logging
+
+from django.db.models.signals import post_save
+
+from django.core.signals import request_finished
+from django.dispatch import receiver
+
+from django.core.exceptions import ObjectDoesNotExist
+from ssl_cert_tracker.models import TestCertsData, TestCertsHistory
+
+logging.basicConfig(filename='pocProject.log',level=logging.DEBUG)
+
+def Insert_Into_CertsData(json_data):

```

Comment:
the name of this function is not PEP8 compliant. please extend the docstring to provide useful information